include/common/bitdepth.rs

```diff
@@ -422,6 +422,48 @@ where
     }
 }
 
+/// Declare and select a [`BitDepth`]-dependent `extern "C" fn`.
```

Comment:
I'm not sure what select you mean in this context? I think this title needs further high-level description in addition to the arguments. Probably want to describe exactly what the declared function name will be here.

Reply:
Okay, I'll try to make it clearer.  By select I mean choose which function to resolve to.

Reply:
Fixed in 1dffe4c.